api/src/main/java/com/lifegraph/team20/controllers/AuthController.java

```diff
@@ -49,14 +54,17 @@
 	@Autowired
 	PasswordEncoder encoder;
 
+	@Autowired
+	AccountRepository accountRepository;
```

Comment:
基本的にアクセス修飾子はつけるようにしてください。
```suggestion
	private AccountRepository accountRepository;
```

Reply:
これは全てのインスタンス変数、メソッドに当てはまることです。